src/model/entity/level.go

```diff
@@ -1,8 +1,76 @@
 package entity
 
+import (
+	"errors"
+	"math/rand"
+)
+
+const (
+	RoomsCount    = 9
+	RoomMinWidth  = 3
+	RoomMinHeight = 3
+	RoomWalls     = 2
+)
+
 type Level struct {
 	Rooms       []Room
 	Passages    []Passage
 	LevelNumber uint
 	LevelEnd    Box
```

Comment:
Вот тут можно упростить, т.к. уже в структуре Level находится:

```suggestion
Number uint
End    Box
```